src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let filteredMen = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    filteredMen = filteredMen.filter(
+      person => Math.ceil(person.died / 100) === century
+    );
+  }
```

Comment:
You can handle it in one filter's callback. For example: `people.filter(person => century ? person.sex === 'm' && getCentury(person.died) === century : person.sex === 'm')`